cpo-docker/README.md

```diff
@@ -0,0 +1,60 @@
+# CPO fullstack docker :whale:
```

Comment:
Is it worth updating/simplifying the _Note_ in the _Functional tests_ section of the main readme by pointing it to this more detailed document on how to run a CPO environment?

Reply:
that's a good point. will update.